plugins/node/opentelemetry-instrumentation-net/package.json

```
@@ -59,7 +59,7 @@
   },
   "dependencies": {
     "@opentelemetry/instrumentation": "^0.51.0",
-    "@opentelemetry/semantic-conventions": "^1.0.0"
+    "@opentelemetry/semantic-conventions": "^1.23.0"
```

nit:

```suggestion
    "@opentelemetry/semantic-conventions": "^1.22.0"
```

Thought perhaps I'm wrong and 1.23 is fine.

---

Same comment as the other PR, I decided to get the latest version, since it also works